.yamllint.yaml

```diff
@@ -0,0 +1,49 @@
+---
+
+yaml-files:
+  - '*.yaml'
+  - '*.yml'
+
+ignore:
+  - deploy/helm-chart/templates/
+
+rules:
+  braces: enable
+  brackets: enable
+  colons: enable
+  commas: enable
+  comments:
+    require-starting-space: true
+    ignore-shebangs: true
+    min-spaces-from-content: 1
+  comments-indentation: enable
+  document-end: disable
+  document-start: disable
+  empty-lines: enable
+  empty-values: enable
+  float-values: disable
+  hyphens: enable
+  indentation:
+    spaces: consistent
+    indent-sequences: consistent
+    check-multi-line-strings: true
+    ignore: |
+      deploy/manifests/nginx-conf.yaml
+      deploy/manifests/njs-modules.yaml
```

Comment:
How will this work with the combined manifest change that's incoming?

Reply:
I've merged the other PR and updated this file to reflect the changes 👍🏼